source/classes.tex

```diff
@@ -6755,7 +6755,7 @@
 
 \pnum
 Access to the deallocation function is checked statically,
-even though a different one might actually be executed.
+even if a different one is actually be executed.
```

Comment:
"is actually be" is broken

Reply:
That was my mistake when merging, fixed.